plugins/legacy-preset-chart-nvd3/src/LineMulti/controlPanel.ts

```diff
@@ -149,10 +149,6 @@ export default {
     y_axis_format: {
       label: t('Left Axis Format'),
     },
-    x_axis_format: {
-      choices: D3_TIME_FORMAT_OPTIONS,
-      default: 'smart_date',
-    },
```

Comment:
This one appears to have the same options as the default for that control, so it's unneeded.